File: Presentation/Models/HttpModels/Requests/PaymentUpdateStatusRequestDto.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using CosmeticShop.Domain.Enumerations;

namespace HttpModels.Requests
{
    public class PaymentUpdateStatusRequestDto
    {
        public Guid OrderId { get; set; }
        [Required]
        public PaymentStatus NewPaymentStatus { get; set; }
    }
}

File: Presentation/cosmeticshop.client/src/apiClient/models/payment-update-status-request-dto.ts
import { OrderStatus } from './order-status';
 /**
 * 
 *
 * @export
 * @interface PaymentUpdateStatusRequestDto
 */
export interface PaymentUpdateStatusRequestDto {

    /**
     * @type {string}
     * @memberof PaymentUpdateStatusRequestDto
     */
    orderId: string;

    /**
     * @type {OrderStatus}
     * @memberof PaymentUpdateStatusRequestDto
     */
    newPaymentStatus: OrderStatus;
}